src/test/java/GreetzLoginFavouritesTest.java

```diff
@@ -0,0 +1,134 @@
+    }
+
+    private void testLoginImpl() throws InterruptedException {
+        driver.get("https://www.greetz.nl/auth/login");
+        Thread.sleep(4000);
+
+        WebElement loginFormElem = driver.findElement(By.id("loginForm"));
+        String email = "test_testing@greetz.com";
+        String pass = "testgreetz94";
+        loginFormElem.findElement(By.name("email")).sendKeys(email);
+        loginFormElem.findElement(By.name("password")).sendKeys(pass);
+        Thread.sleep(3000);
+        driver.findElement(By.id("login-cta")).click();
+        Thread.sleep(5000);
+        Assert.assertTrue(isLoggedIn("test"), "Failed to log in user " + email);
+        Thread.sleep(5000);
+    }
+
+    public boolean isLoggedIn(String firstName) {
+        try {
+            driver.findElement(By.xpath("//div[@class='header-message']//span[text()='Welkom " + firstName + "']"));
+            return true;
+        } catch (NoSuchElementException e) {
+            return false;
+        }
+    }
+
+    @Test
+    public void testGreetzLogin() throws InterruptedException {
+        driver = createDriverInstance();
+        try {
+            testLoginImpl();
+        } finally {
+            driver.quit();
+        }
+    }
+
+    @Test
+    public void testFavouritesFunctionality() throws InterruptedException {
+        driver = createDriverInstance();
+        try {
+            testLoginImpl();
+
+            driver.get("https://www.greetz.nl/ballonnen/denken-aan");
+            Thread.sleep(3000);
+
+            WebElement chosenItem = driver.findElement(By.xpath("//div[text()=\"Ballon 'op fleuren'\"]/ancestor::a/following-sibling::a/div"));
+            chosenItem.click();
+            String expectedItemTitle = driver.findElement(By.xpath("//a[@href='/ballonnen/detail/ballon--op-fleuren-/1142785578']/div[2]")).getText();
+            String expectedItemPrice = driver.findElement(By.xpath("//div[text()=\"Ballon 'op fleuren'\"]/ancestor::a//span/span")).getText();
+            Thread.sleep(3000);
+
+            WebElement myGreetz = driver.findElement(By.linkText("MyGreetz"));
+            myGreetz.click();
+            Thread.sleep(3000);
+
+            WebElement favorieten = driver.findElement(By.xpath("//div[@class=\"hamburgermenu profilehamburgermenu hamburgermenu_visible\"]//span[text()='Favorieten']"));
+            favorieten.click();
+            Thread.sleep(3000);
+
+            WebElement favItem = driver.findElement(By.xpath("//*[@class='favorite-item']"));
+            favItem.click();
+            Thread.sleep(3000);
+
+            String actualItemTitle = driver.findElement(By.xpath("//*[@ng-bind=\"gift.title\"]")).getText();
+            String actualItemPrice = driver.findElement(By.xpath("//*[@class=\"price-normal\"]")).getText();
+            Assert.assertTrue(actualItemPrice.contains(expectedItemPrice), "Prices of basket item: " + actualItemPrice + " and of chosen item: " + expectedItemPrice + " are different.");
```

Comment:
Is the `contains` really what you want to achieve? I guess you should check equality

Reply:
Addressed in 7968d48.

Applied Assert.assertEquals().